### PR TITLE
Video Uploads context updates

### PR DESCRIPTION
**Pull request by wowkalucky**
@kabirkhan @saravpal 

PR includes xblock updates related to new Video Upload page flow.

## Changed
- mocked dependencies during unit test running;
- Organization Azure related configuration moved to `azure-video-pipeline` application;
- implemented basic functionality fallback (if `azure-video-pipeline` not available);
- transcript download URL field mutated to transcripts enabled switch;
- course videos are fetched now from platform contentstore;
- videos management studio view tab styled;
- added / updated unit tests;

### CODE REVIEW

File: azure_media_services/static/js/spec/javascript-code-spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Tests for studio edit's utils
+ */
+/* global AzureMediaServicesBlock */
+
+describe('exspression: 1+1', function() {
```
Comment (sdolenc) [inline azure_media_services/static/js/spec/javascript-code-spec.js]: spelling: should be expresssion

**Review by sdolenc — APPROVED** [review]
great work!

**Comment by sdolenc** [comment]
We noticed there are conflicts in this PR. Can you please resolve them?
The current PR may contain changes that were merged earlier.

**Comment by wowkalucky** [comment]
@sdolenc resolved.

**Review by sdolenc — APPROVED** [review]

File: azure_media_services/ams.py [hunk 2]
```diff
@@ -6,20 +7,29 @@
...
+from xblock.fields import Boolean
 from xblock.fragment import Fragment
 from xblockutils.resources import ResourceLoader
 from xblockutils.studio_editable import StudioEditableXBlockMixin
 
-from .media_services_management_client import MediaServicesManagementClient
-from .models import SettingsAzureOrganization
 from .utils import _
 
+APP_AZURE_VIDEO_PIPLINE = True
```
Comment (kabirkhan) [inline azure_media_services/ams.py]: APP_AZURE_VIDEO_PIPLINE => APP_AZURE_VIDEO_PIPELINE (missing the E in pipeline)

File: azure_media_services/static/js/player.js [hunk 3]
```diff
@@ -1,7 +1,46 @@
...
+    played: 'edx.video.played',
+    paused: 'edx.video.paused',
+    stopped: 'edx.video.stopped',
+    positionChanged: 'edx.video.position.changed',
+    transcriptShown: 'edx.video.transcript.show',
+    transcriptsHidden: 'edx.video.transcript.hidden',
+    videoLoaded: 'edx.video.loaded',
+    captionsShown: 'edx.video.closed_captions.shown',
+    captionsHidden: 'edx.video.closed_captions.hidden'
+};
```
Comment (kabirkhan) [inline azure_media_services/static/js/player.js]: Can you capitalize all the keys here? Would be much clearer in the code.

e.g.
var events = {
PLAYED: 'edx.video.played',
...
}

File: azure_media_services/static/js/studio_edit.js [hunk 4]
```diff
@@ -193,6 +195,113 @@ function StudioEditableXBlockMixin(runtime, element) {
...
+        $containerCaptions.empty();
+        if (data.length === 0) {
+            $containerCaptions.text(gettext('No captions/transcripts available for selected video.'));
+        } else {
+            for (i = 0; i < data.length; i++) {
+                html = '<li class="select-holder"><div class="wrap-input-captions"><input id="checkbox-captions-' + i +
+                        '" type="checkbox" name="captions" value="' + data[i].download_url +
+                        '" data-srclang="' + data[i].language + '" data-label="' + data[i].language_title +
+                        '"/><label for="checkbox-captions-' + i + '">' + data[i].file_name +
+                        ' (' + data[i].language + ')</label></div></li>';
```
Comment (kabirkhan) [inline azure_media_services/static/js/studio_edit.js]: We should probably escape dynamic HTML here.

File: azure_media_services/static/js/studio_edit.js [hunk 5]
```diff
@@ -193,6 +195,113 @@ function StudioEditableXBlockMixin(runtime, element) {
...
+        $containerCaptions.html('<div class="loader-wrapper"><span class="loader"><svg class="icon icon-spinner11">' +
+            '<use xlink:href="#icon-spinner11"></use></svg></span></div class="loader-wrapper">');
+        $.ajax({
+            type: 'POST',
+            url: handlerUrlGetCaptionsAndVideoInfo,
+            data: JSON.stringify({edx_video_id: edxVideoID}),
+            dataType: 'json',
+            success: function(data) {
+                if (data.error_message !== '') {
+                    $containerCaptions.html('<span class="ams-info">' + data.error_message + '</span>');
```
Comment (kabirkhan) [inline azure_media_services/static/js/studio_edit.js]: We should address all of these cases. Pretty sure you can use _.escape(dynamic_data) to html escape.
Should be done everywhere there is dynamic html

**Review by kabirkhan — CHANGES_REQUESTED** [review]

**Comment by kabirkhan** [comment]
Mostly minor changes needed. Great work!

**Review by btelnes — APPROVED** [review]

**Review by btelnes — APPROVED** [review]

**Comment by oksana-slu** [comment]
@kabirkhan 
We have fixed all your comments

**Review by kabirkhan — APPROVED** [review]

**Pull request merged**